views/components/ConfigurationsList/index.jsx

```diff
@@ -305,15 +305,32 @@ const ConfigurationsList = ({ user, classes, theme }) => {
         }
         return response.json()
       })
-      .then((response) => {
+      .then(({ data }) => {
         setState((prevState) => {
           return {
             ...prevState,
-            preferences: babyProofPreferences(response),
+            preferences: babyProofPreferences(data.preferences),
```

Comment:
should we be setting the user state object with this data? Also, do we need both the user and preferences in state or can we get preferences from the user?